Core/FieldHandler/EzCountry.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Kaliop\eZMigrationBundle\Core\FieldHandler;
+
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\Core\FieldType\Country\Value as CountryValue;
+use eZ\Publish\Core\FieldType\Selection\Value as SelectionValue;
+use Kaliop\eZMigrationBundle\API\FieldValueImporterInterface;
+
+class EzCountry extends AbstractFieldHandler implements FieldValueImporterInterface
+{
+    private $contentTypeService;
+
+    private $countries;
+
+    public function __construct(ContentTypeService $contentTypeService, array $countries)
+    {
+        $this->contentTypeService = $contentTypeService;
+        $this->countries = $countries;
+    }
+
+    /**
+     * Creates a value object to use as the field value when setting a country field type.
+     *
+     * @param $fieldValue string|string[] should work
+     * @param array $context The context for execution of the current migrations. Contains f.e. the path to the migration
+     * @return CountryValue
+     */
+    public function hashToFieldValue($fieldValue, array $context = array())
+    {
+        if ($fieldValue === null) {
+            return new CountryValue();
+        }
+
+        $fieldSettings = $this->loadFieldSettings($context['contentTypeIdentifier'], $context['fieldIdentifier']);
+
+        $countries = [];
+        if (!$fieldSettings['isMultiple'] && is_string($fieldValue)) {
+            $key = $this->findCountryKeyByValue($this->countries, $fieldValue);
+            $countryKey = array_keys($this->countries)[$key];
+            $countries = [$fieldValue => $this->countries[$countryKey]];
+            return new CountryValue($countries);
+        }
+
+        if (is_string($fieldValue)) {
+            $fieldValue = array($fieldValue);
+        }
+
+        foreach ($fieldValue as $val) {
+            $key = $this->findCountryKeyByValue($this->countries, $val);
+            $countryKey = array_keys($this->countries)[$key];
+            $countries[$countryKey] = $this->countries[$countryKey];
+        }
+
+        return new CountryValue($countries);
+    }
+
+    private function findCountryKeyByValue($countries, $value)
```

Comment:
Instead of returning an 'array key', why not return tha alpha-2 code (which is also the key into `$this->countries`) ?

Reply:
ok to return the aplha-2 let me process your second statement :)

Reply:
about exploding. you mean loop all the countries array from the ez param and create two news array there?

Reply:
Refactored with your suggestions. let's see what travis says.
still the doubt: Should we thrown an exception in case no country is found? currently, for those cases the migration will fail with something like 
```
Migration failed! Reason: Error in execution of step 1: Notice: Undefined offset: 0 in file /var/www/html/project/ezplatform/vendor/kaliop/ezmigrationbundle/Core/FieldHandler/EzCountry.php line 62
```